src/net.gr

```diff
@@ -138,7 +138,7 @@ provide module DNS {
 
   @unsafe
   provide let resolveTimeout = (host: String, timeout: Int64) => {
-    let hostPtr = WasmI32.add(WasmI32.fromGrain(host), 8n)
+    let hostPtr = WasmI32.(+)(WasmI32.fromGrain(host), 8n)
```

Comment:
I think Oscar had said something about this style breaking the formatter or something like that. given `+` doesn't look to be used anywhere else in this function it would probably be best to just use the from syntax here.